docs/pages/button.js

```diff
@@ -259,245 +212,503 @@ card(
+    <MainSection.Subsection
+      title="Width"
+      description={`
+1. Inline (default)
+    Inline is our default Button width.  The width of an inline Button is based on the length of its text. Use in most cases where you need a Button.
+2. Full-width (\`fullWidth\`)
+    Full-width Buttons can be used in narrower content areas when the text in the Button is close to full width in the content area. This is especially common to see in components such as Callout and Upsell at their smaller breakpoints.`}
+    >
+      <CombinationNew fullwidth={[false, true]}>
+        {({ fullwidth }) => (
           <Button
-            color={selectedTransparent ? 'transparentWhiteText' : 'transparent'}
-            iconEnd="add-pin"
-            text="Save this image"
+            accessibilityLabel={`Example width ${fullwidth}`}
+            color="red"
+            text="Save"
+            fullWidth={fullwidth}
+            size="lg"
           />
-          </Box>
-          <Box padding={2} position="absolute" bottom right>
+        )}
+      </CombinationNew>
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Color on white backgrounds"
+      description={`
+1. Red (Primary)
+    High emphasis, used for primary actions.
+2. Blue (Primary in shopping context)
+    The blue Button is only intended for the shopping experience and is used for primary shopping actions.
+3. Gray (Secondary)
+    Medium emphasis, used for secondary actions.
+4. Transparent (Tertiary)
+    Low emphasis when placed on dark/image backgrounds, used for tertiary actions in that context. *Note, this treatment should be used with caution as it has potential color contrast issues.*
+`}
+    >
+      <CombinationNew color={['red', 'blue', 'gray', 'transparent']}>
+        {({ color }) => {
+          const map = { red: 'Save', blue: 'Shop', gray: 'Visit', transparent: 'Learn more' };
+          return (
             <Button
-            color={selectedTransparent ? 'transparent' : 'transparentWhiteText'}
-            iconEnd="pin-hide"
-            text="Hide this image"
+              accessibilityLabel={`Example width ${color}`}
+              color={color}
+              text={map[color]}
+              size="lg"
+            />
+          );
+        }}
+      </CombinationNew>
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      columns={2}
+      title="Color on color/image backgrounds"
+      description={`
+  1. White (Primary)
+      High emphasis when placed on color/image backgrounds, used for primary actions in that context.
+  2. Semi-transparent white (Secondary)
+      Medium emphasis when placed on color/image backgrounds, used for secondary actions in that context.
+`}
+    >
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+function WhiteButtonExample() {
+  const [open, setOpen] = React.useState(false);
+  const anchorRef = React.useRef();
+  const viewRef = React.useRef();
+
+  React.useEffect(() => {
```

Comment:
The useEffect and useState here aren't necessary. Just put the Layer+Popover in the markup with no conditional. The reader doesn't need to be able to close the Popover on click.

Reply:
Strange. When I remove those elements, the Popover doesn't display... Let me keep digging in on this.

Reply:
Chatted w/ Ryan and we're going to keep in for now as removing this logic prevents the popover from displaying.